cbv3_django_prototype/cbv3_django_prototype/resources/tests.py

```diff
@@ -9,10 +9,10 @@
 class ResourcesTests(APITestCase):
 
     def setUp(self):
-
         call_command('loaddata', 'users.json', verbosity=0)
         call_command('loaddata', 'resources.json', verbosity=0)
-        call_command('loaddata', 'tags.json', verbosity=0)
+        call_command('loaddata', 'tagging.json', verbosity=0)
+        call_command('loaddata', 'taggeditems.json', verbosity=0)
```

Comment:
Instead of loading fixtures, I'd recommend either using `pytest` fixtures, a library like `factory_boy` to generate data, or just creating the needed model instances here in the `setUp`. Fixtures get out of date quickly and are fairly hidden. It's also hard to know if you can change them to create new data conditions for tests. This would also solve your problem of needing to fetch items below as you could associate the created objects with the class and then use them in tests below.

Reply:
Thank you for the reminder.  We have #48 out as a specific issue around this...the logic being that I was struggling to get tagging related features done (and wanted at least some very minimal tests to go with them), so bailed on the testing infrastructure in the short term.

I will be adding additional issues for testing models, serializers, and views as well now that the "major" changes are done-ish.